uctMaths/competition/views.py

```diff
@@ -45,88 +46,7 @@ def printer_entry_result(request, school_list=None):
-            responsible_teacher = None
-        if alt_responsible_teacher:
-            alt_responsible_teacher = alt_responsible_teacher[0]
-        else:
-            alt_responsible_teacher = None
-        #If someone managed to get to this page without having made an entry
-        if not (responsible_teacher or alt_responsible_teacher) and not school_list:
-            return HttpResponseRedirect('../students/newstudents.html')
-        #If the school has an entry
-        elif responsible_teacher:
-            c = {'type':'Students',
-                'timestamp':timestamp,
-                'schooln':assigned_school,
-                'delivery_address':assigned_school.address,
-                'contact_phone':assigned_school.phone,
-                'responsible_teacher':responsible_teacher,
-                'alt_responsible_teacher': alt_responsible_teacher,
-                'student_list':individual_list,
-                'pair_list':pair_list,
-                'max_num_pairs':compadmin.admin_number_of_pairs(),
-                'entries_open':compadmin.isOpen() or request.user.is_staff,
-                'invigilator_list': invigilator_list,
-                'grades':range(8,13),
-                'grade_left':range(8,11),
-                'invigilator_range':range(10-len(invigilator_list)), 
-                'igrades':range(8,13),
-                'total_num':int(count_pairs*2+count_individuals),
-                'year':year,
-                'invigilators_required':compadmin.competition_has_invigilator()}
-            #Render the template with the context (from above)
-            template = get_template('printer_entry.html')
-            c.update(csrf(request))
-            context = Context(c)
-            html += template.render(context) #Concatenate each rendered template to the html "string"
-        else:
-            c = {'type':'Students',
-                'timestamp':timestamp,
-                'schooln': assigned_school,
-                'grades':range(8,13),
-                'grade_left':range(8,11),
-                'invigilator_range':range(10-len(invigilator_list)), 
-                'igrades':range(8,13),
-                'max_num_pairs':compadmin.admin_number_of_pairs(),
-                'total_num':'No students entered for this school',
-                'invigilators_required':compadmin.competition_has_invigilator()}
-
-            #Render the template with the context (from above)
-            template = get_template('printer_entry.html')
-            c.update(csrf(request))
-            context = Context(c)
-            html += template.render(context) #Concatenate each rendered template to the html "string"
-
-
-    result = StringIO.StringIO()
-
-    #Generate the pdf doc
-    pdf = pisa.pisaDocument(StringIO.StringIO(html.encode("UTF-8")), result, encoding='UTF-8')
-    if not pdf.err:
-        return result
-    else:
-        pass #Error handling?
+    return compadmin.generate_answer_sheets(request, school_list)
```

Comment:
should `school_list` be `temp_school_list` here?

Reply:
Yeah, sorry about that